packages/react/src/components/form/form.tsx

```diff
@@ -110,6 +111,11 @@ export const Form = ({
           {flowState.status === "authenticating" && (
             <FormProvider>{slots.authenticating}</FormProvider>
           )}
+          {flowState.status === "storeRecoveryCodes" && (
+            <FormProvider>
```

Comment:
Is the `FormProvider` needed here? Looking at `StoreRecoveryCodes` it seems not, but I might be wrong 😄

Reply:
Nope it's not, I'll remove it :)